### PR TITLE
Minimum number of additions and deletions to convert one string to another in C++

### PR DESCRIPTION
**Pull request by SAY-droid427**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

This code implements the minimum number of additions and deletions required to convert one string to other in C++

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
In response to:
https://github.com/TesseractCoding/NeoAlgo/issues/2287#issuecomment-793232282

### CODE REVIEW

File: C-Plus-Plus/dp/min_insert_del.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//Driver Code
+int main()
+{
+    string x,y;
+    cin>>x>>y;
+    //Calculating the length of the string x and y
+    int n=x.length();
+    int m=y.length();
+    minInsertDel(x,y,n,m);
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/min_insert_del.cpp:52]: Add a new line at the EOF

File: C-Plus-Plus/dp/min_insert_del.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+}
+
+
+//Driver Code
+int main()
+{
+    string x,y;
+    cin>>x>>y;
+    //Calculating the length of the string x and y
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/min_insert_del.cpp:48]: Redundant Comment

File: C-Plus-Plus/dp/min_insert_del.cpp [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+using namespace std;
+
+void minInsertDel(string x,string y,int n,int m)
+{
+    //Table
+    int t[n+1][m+1];
+    
+    //Base conditions
+    //For calculating the longest common subsequence
+    //If the length of any one of the string is zero, then the length of common subsequence will be zero
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/min_insert_del.cpp:16]: Use Multi-Line Comments here

File: C-Plus-Plus/dp/min_insert_del.cpp [hunk 4]
```diff
@@ -0,0 +1,52 @@
+//This code computes the total number of insertions and deletions to convert string x into y
+/** CONCEPT
+   To convert x to y, we find out the length of the longest common subsequence and then find out the difference between the length of the first string and the longest common subsequence to find out the number of deletions.
+   To find out the number of insertions we find out the difference between the length of the second string and the longest common subsequence
+**/
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/min_insert_del.cpp:5]: Modularize this and add justified information

File: C-Plus-Plus/dp/min_insert_del.cpp [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+#include<bits/stdc++.h>
+using namespace std;
+
+void minInsertDel(string x,string y,int n,int m)
+{
+    //Table
+    int t[n+1][m+1];
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/min_insert_del.cpp:12]: Use more meaningful variable names

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
None of the Comments are resolved. Let me know when you are done.

**Comment by SAY-droid427** [comment]
I am sorry I have created a separate pull request. By mistake I had added code to this pull request and was unable to push the new changes to this branch. I have created a separate pull request afresh. Sorry for the inconvenience

**Pull request closed without merging**